lib/src/drift_sql_database_server.dart

```diff
@@ -158,6 +158,9 @@ SQLDataType _makeType(
       return SQLDataType.real;
     case DriftSqlType.any:
       return SQLDataType.text;
+    case CustomSqlType<Object>():
```

Comment:
Does this always work will this break (wrt type erasure) with different generics?

Reply:
When implementing `CustomSQLType`, the `mapToSqlLiteral` function returns a `String`, so I think whichever custom type you implement/include, it'll be stored as a String internally (and can be displayed as such in the inspector). 

https://drift.simonbinder.eu/docs/sql-api/types/

Of course, ideally, you'd want to account for different types from other DB systems by displaying them properly already, such as a parsed time or number for instance, but these are extensions onto Drift, not part of the package itself, so I don't think it'd be worth it to go through a lot of effort to support them natively in the inspector.